app/src/main/java/com/kylecorry/trail_sense/astronomy/infrastructure/AstronomyPreferences.kt

```diff
@@ -16,7 +16,7 @@ class AstronomyPreferences(private val context: Context) {
 
```

Comment:
Can you undo the changes to this file? These are preference keys, not strings - therefore it needs to be context.getString